### PR TITLE
Removed the deprecated static helpers.

### PR DESCRIPTION
**Pull request by natcarey**

### CODE REVIEW

**Review by matthewmascord — COMMENTED** [review]
One of the ACs was for the version number to be increased to 1.0.0.

I think the play-frontend-govuk changes need to be made first, otherwise we'll end up with only half of the static helpers being removed.

Could we squash commits please?

File: src/it/scala/uk/gov/hmrc/hmrcfrontend/support/TemplateIntegrationSpec.scala [hunk 1]
```diff
@@ -39,6 +45,8 @@ abstract class TemplateIntegrationSpec[T: OWrites: Arbitrary](hmrcComponentName:
   override def overrideParameters(p: Test.Parameters): Test.Parameters =
     p.withMinSuccessfulTests(20)
 
+  val component = app.injector.instanceOf[C]
```
Comment (oscarduignan) [inline src/it/scala/uk/gov/hmrc/hmrcfrontend/support/TemplateIntegrationSpec.scala:48]: @natcarey with this refactoring could we now move the render method to the abstract class too? Seems like it's now basically the same code except for the type of the render param but we have that in T

Comment (natcarey) [reply src/it/scala/uk/gov/hmrc/hmrcfrontend/support/TemplateIntegrationSpec.scala:48]: Ooh, that would be lovely.  I'll try that. :)

**Comment by natcarey** [comment]
Yeah, I'm holding off the 1.0.0 until we've merged `play-frontend-govuk` in - that's particularly important if we haven't ruled out a name change.

File: docs/maintainers/adr/0013-removing-deprecated-helpers.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# play-frontend-hmrc mirrors hmrc-frontend
```
Comment (matthewmascord) [inline docs/maintainers/adr/0013-removing-deprecated-helpers.md:1]: Title needs updating.

File: docs/maintainers/adr/0013-removing-deprecated-helpers.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+* Status: accepted
+* Date: 2020-06-24
```
Comment (matthewmascord) [inline docs/maintainers/adr/0013-removing-deprecated-helpers.md:4]: Date needs updating.

Comment (natcarey) [reply docs/maintainers/adr/0013-removing-deprecated-helpers.md:4]: The date is based on the comment in Jira documenting the decision when we made it.

File: docs/maintainers/adr/0013-removing-deprecated-helpers.md (hunk 2 above)
Comment (natcarey) [reply docs/maintainers/adr/0013-removing-deprecated-helpers.md]: Updated.

**Comment by matthewmascord** [comment]
Superceded by https://github.com/hmrc/play-frontend-hmrc/pull/112

**Pull request closed without merging**